Feature/twitch chat

thomas-pipithkul opened the pull request:
1 minor inconvenient: user need to log in to twitch player & twitch chat components separately because embed Twitch is not well supported by Twitch.

thomas-pipithkul committed:
add new toggle for the whole side bar & different expand/collapse side bar icon

thomas-pipithkul committed:
collapsible chat

thomas-pipithkul committed:
add chat for mobile

thomas-pipithkul committed:
Merge remote-tracking branch 'origin/dev' into feature/twitch-chat

MihaZupan commented on src/components/Progress/Goals.tsx:
```diff
@@ -0,0 +1,43 @@
+              game={value.name} 
+              rank={value.info.rankGoal} 
+              icon={value.info.rankGoalIcon} 
+              progress={getPercent(value.info.currentRankIndex, value.info.goalRankIndex)}
```
```suggestion
              progress={getPercent(value.info.currentRankIndex + (value.info.pointsInRank / 100), value.info.goalRankIndex)}
```

thomas-pipithkul committed:
Update src/components/Progress/Goals.tsx

Good suggestion! rank index + points% is better representation of overall progress

Co-authored-by: Miha Zupan <mihazupan.zupan1@gmail.com>
